docs/field-details.asciidoc

```diff
@@ -1908,6 +1908,84 @@ example: `ipv4`
 
 |=====
 
+[[ecs-nodejs]]
+=== Node.js Fields
+
+Fields describing a Node.js runtime.
+
+These fields can be used for monitoring the performance of a Node.js application.
+
+==== Node.js Field Details
+
+[options="header"]
+|=====
+| Field  | Description | Level
+
+// ===============================================================
+
+| nodejs.eventloop.delay.avg
+| The average event loop delay for the reporting period.
+
+Event loop delay is sampled every 10 milliseconds. Delays shorter than 10 milliseconds may not be observed, for example if a blocking operation starts and ends within the same sampling period.
```

Comment:
FYI: The "10 milliseconds" is a configurable number. In the Elastic APM Node.js agent we have currently hard-coded it to 10 ms (which is also the default in Node.js), but the Node.js API we use the extract this information can be configured to any other duration.

Should we make a PR to update this? And if so, what should we write instead?

Reply:
Ah yeah I was perhaps being a bit too specific about how the APM agent implementation works. Yes we should probably update this.

How about this?

> Event loop delay is periodically sampled, e.g. every 10 milliseconds, but this may vary by source. Delays shorter than the sampling period may not be observed, for example if a blocking operation starts and ends within the same sampling period.